src/client/java/com/convallyria/taleofkingdoms/client/gui/entity/GuildMasterScreen.java

```diff
@@ -117,8 +117,16 @@ public void init() {
         final ButtonWidget fixWidget = this.addDrawableChild(ButtonWidget.builder(Text.translatable("menu.taleofkingdoms.guild_master.fix_guild"), widget -> {
             final TaleOfKingdomsAPI api = TaleOfKingdoms.getAPI();
             api.executeOnMain(() -> {
-                if (instance.isUnderAttack() || guildPlayer.getCoins() < 3000) return;
-                if (stack == null) return;
+                if (instance.isUnderAttack()) {
+                    Translations.GUILDMASTER_UNDER_ATTACK.send(player);
+                    return;
+                }
```

Comment:
In reality, we have two guild master entities: The normal guild master, and the "under attack" guild master. You can only repair the guild with the "under attack" guild master. So I'm not sure if this text should exist.

Reply:
Thank you for pointing this out. I've removed this check and the corresponding translation for GUILDMASTER_UNDER_ATTACK. :)